### PR TITLE
[9.0][ADD] website_version_ce: Website Versioning

### PR DESCRIPTION
**Pull request by lasley**
Supersedes #242.

[This PR comment has not yet been attended to](https://github.com/OCA/website/pull/242#discussion_r126432113). All others have.

### CODE REVIEW

**Comment by lasley** [comment]
Superseded by #379

**Comment by astirpe** [comment]
Hi @lasley, my PR was not meant to supersede this one, which remains valid in my opinion. You could leave this PR open... or maybe you wanted to avoid duplication of effort and focus on one PR at a time?

**Comment by lasley** [comment]
@astirpe - Oh I'm dumb. Your's is in v10 and mine is v9. For some reason I thought mine was v10 too. Reopened

File: website_version_ce/models/version.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            version.view_ids.publish()
+
+    @api.multi
+    def publish_version(self, save_master, copy_master_name):
+        self.ensure_one()
+        del_l = self.env['ir.ui.view']
+        copy_l = self.env['ir.ui.view']
+        ir_ui_view = self.env['ir.ui.view']
+        for view in self.view_ids:
+            master_id = ir_ui_view .search([
```
Comment (astirpe) [inline website_version_ce/models/version.py:59]: There is a blank between `ir_ui_view` and `.search`

**Comment by simahawk** [comment]
@astirpe shall we keep this open?

**Comment by astirpe** [comment]
Can be closed for me. Thanks!

**Comment by simahawk** [comment]
good, thanks!
we can reopen it if needed as usual.

**Pull request closed without merging**